pkg/controller/podgc/gc_controller_test.go

```diff
@@ -657,6 +662,181 @@ func TestGCTerminating(t *testing.T) {
 	testDeletingPodsMetrics(t, 7)
 }
 
+func TestGCDeleteAllPodsBoundToNode(t *testing.T) {
+	testNode := &v1.Node{
+		ObjectMeta: metav1.ObjectMeta{Name: "testNode"},
+		Status: v1.NodeStatus{
+			Capacity: v1.ResourceList{
+				v1.ResourceName(v1.ResourceCPU):    resource.MustParse("10"),
```

Comment:
Do you need the capacity? Try to minimize the objects to only what is needed.

Reply:
Removed

Reply:
Feel free to resolve